posthog/temporal/data_modeling/run_workflow.py

```diff
@@ -727,6 +728,12 @@ async def run(self, inputs: RunWorkflowInputs) -> Results:
         )
         completed, failed, ancestor_failed = results
 
+        # publish metrics
+        if failed or ancestor_failed:
```

Comment:
Not sure if we want to group these two together. I think I would rather have separate statuses?

Reply:
yeah I wasn't sure what we want here. Guess the extra granularity from having an additional status can't hurt 👍